### PR TITLE
fix: show message when loading templates

### PR DESCRIPTION
**Pull request by eventualbuddha**
Without this, it appears to simply hang on the last template that was uploaded, which feels a little broken. With this change, it's better though still not great.

### CODE REVIEW

File: apps/bsd/src/screens/LoadElectionScreen.tsx [hunk 1]
```diff
@@ -25,6 +25,7 @@ const LoadElectionScreen: React.FC<Props> = ({ setElection }) => {
     isLiveMode: boolean
     locales?: string
   }>()
+  const [isLoadingTemplates, setLoadingTemplates] = useState(false)
```
Comment (beausmith) [inline apps/bsd/src/screens/LoadElectionScreen.tsx:28]: `setIsLoadingTemplates`

(sorry, on mobile)

**Review by beausmith — APPROVED** [review]

File: apps/bsd/src/screens/LoadElectionScreen.tsx (hunk 1 above)
Comment (eventualbuddha) [reply apps/bsd/src/screens/LoadElectionScreen.tsx:28]: I actually disagree with this. In my experience, getters and setters for boolean values typically follow the `isFoo`/`setFoo` pattern. I have had a hard time finding examples for or against this with React hooks specifically (here's [`setToggled`](https://dmitripavlutin.com/react-usestate-hook-guide/) and here's [`setIsToggled`](https://dev.to/alexkhismatulin/update-boolean-state-right-with-react-hooks-3k2i)), but in other contexts like Java [the idiom is quite strongly `setFoo`](https://stackoverflow.com/questions/5322648/for-a-boolean-field-what-is-the-naming-convention-for-its-getter-setter).

As with most conventions, I'm okay with either as long as we're consistent. Have a preference, @benadida & @carolinemodic?

Comment (benadida) [reply apps/bsd/src/screens/LoadElectionScreen.tsx:28]: My sense is the convention we've been using leads to setIsLoadingTemplates, as in the setter is set + gettername. But I do find it a bit odd and I would support the other approach. 

I'm also not one who feels particularly strong about strict naming conventions. So how's that for an unhelpful comment? :)

Comment (eventualbuddha) [reply apps/bsd/src/screens/LoadElectionScreen.tsx:28]: Merged this as-is for now, but let's revisit naming at some point.

**Pull request merged**